master/internal/api_command.go

```diff
@@ -184,6 +184,12 @@ func (a *apiServer) getCommandLaunchParams(ctx context.Context, req *protoComman
 				"cannot set work_dir and context directory at the same time")
 		}
 		config.WorkDir = nil
+
+		modelDef, err = archive.ToTarGz(userFiles)
```

Comment:
can we call this `contextDirectory` or something, instead of model def? here and elsewhere,